DependencyInjection/Configuration.php

```diff
@@ -84,6 +84,19 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->info('When true, resolving of Doctrine mapping is done automatically to use either ORM or ODM object manager')
                     ->defaultTrue()
                 ->end()
+                ->arrayNode('cookie')
+                    ->canBeEnabled()
+                    ->children()
+                        ->enumNode('sameSite')
```

Comment:
I suggest using snake_case for option names for consistency

Reply:
Thanks @chalasr - I've updated the PR.